test/integration/open-with/OpenWith.int.js

```diff
@@ -0,0 +1,22 @@
+// <reference types="Cypress" />
+
+describe('OpenWith', () => {
+    beforeEach(() => {
+        cy.server();
+    });
+
+    it('integrations will render when the dropdown is clicked', () => {
+        cy.route(
+            'GET',
+            '**/files/*/open_with_integrations',
+            'fixture:open_with_integrations/multipleIntegrations.json',
+        );
+
+        cy.visit('/ContentOpenWith');
+
+        // // Click the Open With button
+        cy.getByTestId('multipleintegrationsbutton').click();
+        // // The dropdown should render
+        cy.getByTestId('openwithdropdown').should('exist');
```

Comment:
It might be more meaningful to test the content of the dropdown rather than the dropdown itself via a testid.

Reply:
I've heard that using css selectors is an antipattern with cypress. Since these menu items are dynamically generated, do you have any recommendations for selecting content?

Reply:
`cy.contains(l("translation.id.xyz"))`

Using the i18n util to translate a content string

Reply:
I'll check the strings directly